reana-cluster-dev.yaml

```diff
@@ -0,0 +1,73 @@
+cluster:
+  type: "kubernetes"
+  config: "./development-kubeconfig.yaml"
+  config_context: "minikube"
+  version: "v1.6.4"
+  url: "http://localhost"
+
+components:
+  reana-workflow-controller:
+    type: "docker"
+    image: "reanahub/reana-workflow-controller:v0.0.1"
+    mountpoints:
+      - type: hostPath
+        name: devcomponents
+        path: "/devcomponents/reana-workflow-controller:/code/reana-workflow-controller"
+    environment:
+      - SHARED_VOLUME_PATH: "/reana"
+      - ORGANIZATIONS: "default,alice,atlas,cms,lhcb"
+
+  reana-job-controller:
+    type: "docker"
+    image: "reanahub/reana-job-controller:v0.0.1"
+    mountpoints:
+      - type: hostPath
+        name: devcomponents
+        path: "/devcomponents/reana-job-controller:/code/reana-job-controller"
+    environment:
+      - REANA_STORAGE_BACKEND: "LOCAL"
+
+  reana-server:
+    type: "docker"
+    image: "reanahub/reana-server:v0.0.1"
+    mountpoints:
+      - type: hostPath
+        name: devcomponents
+        path: "/devcomponents/reana-server:/code/reana-server"
+    environment:
+      - API_HOST: "api"
```

Comment:
who is consuming this variable?

Reply:
Hmm, I think that is a leftover from summer when we wanted to specify what could be specified with `reana-cluster.yaml`. There is no functionality to actually propagate envvars from `reana-cluster.yaml`.